### DISCUSSION TITLE
JetBrains.ReSharper.GlobalTools cross platform InspectCode sunset dupfinder

### DISCUSSION THREAD

**Discussion by AdaskoTheBeAsT** [cake-build/cake#3801] [Ideas]
Hi,

could you please consider changing InspectCode cake tool to use cross platform JetBrains package?
[Command line tools cross platform](https://blog.jetbrains.com/dotnet/2020/07/16/resharper-command-line-tools-cross-platform-and-global-tools/)

Please be also aware that unfortunately DupFinder is being sunset
[Sunsetting DupFinder](https://blog.jetbrains.com/dotnet/2021/08/12/sunsetting-dupfinder-command-line-tool/)

**Comment by gep13** [comment]
@AdaskoTheBeAsT I think the best idea here would be to create a new Cake Addin that makes use of this tool, and then we can think about deprecate the aliases that ship within Cake itself.

Would creating an addin for this tool be something that you would be interested in doing?

**Reply by AdaskoTheBeAsT** [reply]
Hi @gep13
Yes I will take that opportunity to contribute :)

**Reply by gep13** [reply]
@AdaskoTheBeAsT that is great news! I can create a repository in the Cake-Contrib Organisation for the addin of the development and provide you access to it.  What do you think would be a good name for the addin?  Normally we do something like Cake.Npm, or Cake.Docker, etc.  However, Cake.JetBrains.ReSharper.GlobalTools seems a little long winded :smile:

**Reply by AdaskoTheBeAsT** [reply]
@gep13 I thought about  Cake.ReSharper.GlobalTools :)

**Reply by gep13** [reply]
Sounds good to me! I have created a repository https://github.com/cake-contrib/Cake.ReSharper.GlobalTools and given you access.  Let me know if you have any questions.

**Reply by AdaskoTheBeAsT** [reply]
Thanks a lot! I will setup basic things - lets see how it will go
